package.json

```diff
@@ -1,43 +1,28 @@
 {
+  "private": true,
   "scripts": {
-    "lerna": "lerna",
-    "postinstall": "lerna bootstrap",
-    "rebootstrap": "lerna clean --yes && lerna bootstrap",
-    "add-package": "addpackage() { mkdir packages/$1 ; pushd packages/$1 ; npm init -y --scope=financial-times ; popd ; } ; addpackage",
-    "build": "athloi build",
-    "pretest": "npm run build",
+    "postinstall": "athloi exec npm i",
+    "clean": "git clean -fxdi",
+    "build": "athloi run build",
     "jest": "jest -c jest.config.js",
     "test": "npm run lint && npm run jest",
     "lint": "eslint . --ext=js,jsx",
-    "start": "athloi",
     "precommit": "secret-squirrel",
     "commitmsg": "secret-squirrel-commitmsg",
-    "postcheckout": "athloi cleanup"
+    "run-workbench": "(cd tools/x-workbench && npm start)",
```

Comment:
something i built for a side project last night, but would be useful here is `@quarterto/open-once-listening` which would let you do

```
"prerun-workbench": "open-once-listening http://local.ft.com:9001 &",
"prerun-docs": "open-once-listening http://local.ft.com:8080 &",
```